Week1/homework.md

```diff
@@ -0,0 +1,29 @@
+# Homework QA Training Week 1: Foundation to Software testing
+
+## Describe shared testing activities for web and mobile
+Responsiveness
+GUI-Frontend
+Sign in/Sign up(Username,email,password check)
+Settings(editing profile)
+CRUD Articles
+CR*D Comments on articles
+GET and display paginated lists of articles
+Favorite articles(Like&Dislike)
+Follow other users
+Routing-URL 
+
+## Describe unique testing activities per platform (Web)
+
+Testing at different browsers
+Testing at different OS(Mac,Linux,Windows ...)
```

Comment:
testing on different OS would make sense it case of desktop applications. But for web applications, browsers are where the application would run so if there is an OS specific issue that would fall onto the browser and not your web app (and yes Chrome, Safari and other browsers can be buggy 😄 )